gradlew

```diff
@@ -160,6 +160,9 @@ save () {
     echo " "
 }
 APP_ARGS=$(save "$@")
+# Realm's work-around for a bug in Gradle 4.1
```

Comment:
Would be nice to have a reference to a bug report somewhere, so we can know when to remove this again